plugins/node/opentelemetry-plugin-mysql/src/mysql.ts

```diff
@@ -16,22 +16,19 @@
 
 import { BasePlugin, isWrapped } from '@opentelemetry/core';
 import { CanonicalCode, Span, SpanKind } from '@opentelemetry/api';
-import * as mysqlTypes from 'mysql';
+import type * as mysqlTypes from 'mysql';
 import * as shimmer from 'shimmer';
-import { AttributeNames } from './enums';
-import { getConnectionAttributes, getSpanName } from './utils';
+import { getConnectionAttributes, getDbStatement } from './utils';
 import { VERSION } from './version';
+import { DatabaseAttribute } from '@opentelemetry/semantic-conventions';
 
 export class MysqlPlugin extends BasePlugin<typeof mysqlTypes> {
   readonly supportedVersions = ['2.*'];
 
   static readonly COMPONENT = 'mysql';
-  static readonly DB_TYPE = 'sql';
 
   static readonly COMMON_ATTRIBUTES = {
-    [AttributeNames.COMPONENT]: MysqlPlugin.COMPONENT,
-    [AttributeNames.DB_TYPE]: MysqlPlugin.DB_TYPE,
-    [AttributeNames.PEER_SERVICE]: MysqlPlugin.COMPONENT,
+    [DatabaseAttribute.DB_SYSTEM]: MysqlPlugin.COMPONENT,
```

Comment:
I think this [is deprecated](https://github.com/open-telemetry/opentelemetry-specification/blob/813aa36d502006f2462eb16244f84fc459c3eb41/CHANGELOG.md#v030-02-21-2020)

Reply:
The `component` attribute is deprecated. This is this `db.system` attribute which is [alive and well])https://github.com/dynatrace-oss-contrib/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#connection-level-attributes)

Reply:
I think the confusion arises from `MysqlPlugin` having a `COMPONENT` property that is used as the value of the attribute as well as lower down as the `moduleName` value for the plugin.

Should we change the prop name? or get rid of it altogether since `db.system` attribute and `moduleName` are separate entities, that so happen to often share the same name but not always (e.g. redis db system for ioredis module)

Reply:
oops my mistake, misread component vs system